pkg/server/storage_api/ranges_test.go

```diff
@@ -143,29 +141,55 @@ func TestRangeResponse(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// TODO(#110018): grant a special capability to the secondary tenant
-	// before the endpoint can be accessed.
+	var isSystemTenant bool = !srv.StartedDefaultTestTenant()
+	t.Run("access range endpoint within current tenant's scope", func(t *testing.T) {
+		var currentTenantRangeId int64
+		// create a new table from the current tenant
+		r := sqlutils.MakeSQLRunner(ts.SQLConn(t))
+		r.Exec(t, "CREATE TABLE t (x INT PRIMARY KEY, xsquared INT)")
+		for i := 0; i < 5; i++ {
+			r.Exec(t, fmt.Sprintf("ALTER TABLE t SPLIT AT VALUES (%d)", 100*i/5))
+		}
+		// get some ranges from the given table
+		r.QueryRow(t, "SELECT range_id FROM [SHOW RANGES FROM TABLE t WITH DETAILS] LIMIT 1;").Scan(&currentTenantRangeId)
+		t.Logf("currentTenantRangeId : %v", currentTenantRangeId)
+		require.NoError(t, validateRangeBelongsToTenant(ts, currentTenantRangeId))
+	})
 
+	t.Run("access range endpoint within system tenant's scope", func(t *testing.T) {
+		if isSystemTenant {
```

Comment:
nit: `isSystemTenant` is only used here, so no need to create a variable for this?